GitAssignment/src/com/company/Main.java

```diff
@@ -3,6 +3,28 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+	public static void Distinct(int []arr) {
```

Comment:
Moataz, 
First of all, good job, but there are some notes to be fixed, your code exceeds the complextiy limit O(n^2) and  you **have to** move your function from the main method, kindly follow this hierarchy:
```
public class Main{
    public static void main (String[] args) {
        // Leave the body of main blank "empty"
    } 

    public static void Distinct(int []arr) {
        // your code goes here
    }
}
```